### PR TITLE
Plumbing device handles deeper into command buffer validation.

### PR DESCRIPTION
**Pull request by benvanik**

### CODE REVIEW

File: iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp [hunk 1]
```diff
@@ -55,8 +55,8 @@ static FailureOr<LinalgTilingAndFusionOptions> getTileAndFuseOptionsFromConfig(
     for (auto op : computeOps) {
       if (auto currLoweringConfig = iree_compiler::getLoweringConfig(op)) {
         if (loweringConfig) {
-          return {funcOp.emitOpError(
-              "unhandled multiple lowering configurations in compute ops")};
+          return LogicalResult(funcOp.emitOpError(
```
Comment (GMNGeoffrey) [inline iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp:58]: Is this related?

**Review by GMNGeoffrey — APPROVED** [review]
Does this fix the cuda failure?

**Comment by GMNGeoffrey** [comment]
Argh automerge :-/ NBD here, but I really wish there was an "approve but not approve automerge"

File: iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp (hunk 1 above)
Comment (benvanik) [reply iree/compiler/Codegen/Sandbox/LinalgTensorCodegenDriver.cpp:58]: was a new warning that snuck in recently

**Comment by benvanik** [comment]
Fixed it locally, but the bots only run it post-submit so 🤞

**Comment by GMNGeoffrey** [comment]
> Fixed it locally, but the bots only run it post-submit so crossed_fingers

We do have the ability to trigger them presubmit FYI

**Comment by benvanik** [comment]
how? (I was looking for a label but didn't spot one)

**Comment by GMNGeoffrey** [comment]
> how? (I was looking for a label but didn't spot one)

It's a button on an internal UI
